<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Slicer</class>
 <widget class="QDialog" name="Slicer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>257</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="imageFileLabel">
     <property name="text">
      <string>I&amp;mage File</string>
     </property>
     <property name="buddy">
      <cstring>txtImageFile</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="txtImageFile"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnOpenImageFile">
       <property name="text">
        <string>Open...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblPieceCount">
     <property name="text">
      <string>&amp;Piece Count:</string>
     </property>
     <property name="buddy">
      <cstring>pc_20</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="pieceCount" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="pc_20">
        <property name="text">
         <string>&amp;20</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="pc_50">
        <property name="text">
         <string>&amp;50</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="pc_100">
        <property name="text">
         <string>&amp;100</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="pc_200">
        <property name="text">
         <string>2&amp;00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="pc_500">
        <property name="text">
         <string>500</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="pc_custom">
        <property name="text">
         <string>&amp;ca.:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="pc_box">
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="gridTypeLabel">
     <property name="text">
      <string>&amp;Grid type:</string>
     </property>
     <property name="buddy">
      <cstring>cboGridType</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cboGridType">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="saveToLabel">
     <property name="text">
      <string>Save to:</string>
     </property>
     <property name="buddy">
      <cstring>txtSaveTo</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="txtSaveTo"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSaveTo">
       <property name="text">
        <string>Open...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Slicer</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Slicer</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
